.github/workflows/lint-test-chart.yaml

```diff
@@ -0,0 +1,44 @@
+name: Lint and Test Chart
```

Comment:
For workflows, we may want to define a naming convention for `.yml` files:

```bash
build-api.yml         # run build, unit tests, linter
build-client-java.yml # run build, unit tests, linter
test-helm.yml         # run linter, test against kind k8s cluster
...
```

That is, I would take a general-to-more-specific naming convention `[action]-[group]-[module]`, where `[group]` is optional. It's a similar approach we took for naming CI jobs in CircleCI. But, that might result in a growing number of workflows (well, maybe not, `< 10`).


We could also just want to map workflows to Marquez's sub-projects defined 
```
api.yml
web.yml
clients.yml
spec.yml
chart.yml
```

Personally, it might be better to have more discrete workflows, then define them per sub-projects. Thoughts?

Reply:
Great idea! I like the option of combining the Marquez sub-module with the action being taken. How about this for the file in this PR?

_test-chart.yml_

If we were to extend the workflows to include Helm chart releasing, then we could add another file that would be called _release-chart.yml_. Not planning that right now, just wanted to test how the nomenclature could look.

Reply:
Yeah, let's use `test-chart.yml`, but with the assumption that things will change. I need to review the docs for GitHub Actions as well. I'm thinking we might have:

```
.github/workflows/
  ci.yml
  release.yml
```
as a way to further simply our workflows. Just a thought of course, but will have a proper discussion when we begin to replace CircleCI with GitHub Actions.